packages/api/src/services/library_item.ts

```diff
@@ -238,10 +239,16 @@ const buildWhereClause = (
     args.matchFilters.forEach((filter) => {
       const param = `match_${filter.field}`
       queryBuilder.andWhere(
-        `websearch_to_tsquery('english', :${param}) @@ library_item.${filter.field}_tsv`,
-        {
-          [param]: filter.value,
-        }
+        new Brackets((qb) => {
+          qb.andWhere(
+            `websearch_to_tsquery('english', :${param}) @@ library_item.${filter.field}_tsv`,
+            {
+              [param]: filter.value,
+            }
+          ).orWhere(`${filter.field} ILIKE :value`, {
+            value: `%${filter.value}%`,
```

Comment:
i'm wondering if just prefix matching is enough for us like`${filter.value}%`?

Reply:
yeah i think you are right. Updating

Reply:
oh actially, i wonder if prefix and suffix is needed for items like author `author:Harper` for `author:Jackson Harper`

Reply:
yeah, you are right. we should do both prefix and suffix.

thank you!